yarn.lock

```diff
@@ -0,0 +1,372 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
No objection to including a yarn.lock, but we should probably also include a package-lock.json just for consistency with the main ably-js repo, and have some process for ensuring that the two stay in sync? wdyt?

Reply:
I added `yarn.lock` here out of my personal preference. This is completely ignored for end-user installs, I can replace it completely with package-lock if you want. I'm fine with either